.github/workflows/homebrew-core-head.yml

```diff
@@ -30,6 +30,8 @@ jobs:
       - name: Brew update
         run: brew update
       - name: Brew Install
+        env:
+          HOMEBREW_NO_INSTALL_FROM_API: 1
```

Comment:
please add a comment for this kind of thing. There's no way another person can know why we need those things.
I can google it, but it's far more efficient if there's a comment before explaining the reasoning.